src/extension.ts

```diff
@@ -60,17 +67,15 @@ function runJavaServer(context: VSCode.ExtensionContext): Thenable<StreamInfo> {
             requirements,
             server.address().port
           );
-          console.log(`Executing ${command} ${args.join(' ')}`);
+          logToSonarLintOutput(`Executing ${command} ${args.join(' ')}`);
           const process = ChildProcess.spawn(command, args);
 
-          if (VERBOSE_LOGS) {
-            process.stdout.on('data', function(data) {
-              console.log(data.toString());
-            });
-            process.stderr.on('data', function(data) {
-              console.error(data.toString());
-            });
-          }
+          process.stdout.on('data', function(data) {
+            logToSonarLintOutput(data.toString());
```

Comment:
Here I'm not sure we should use appendLine, since data will probably contains carriage returns (to be verified)

Reply:
Yup, confirmed with experiment. I also believe that showing the raw `stdout`/`stderr` content (instead of adding newline characters) will make troubleshooting easier.